_includes/footer.html

```diff
@@ -25,12 +25,12 @@
     <span class="screen-reader-text">Mastodon</span>
 </a>{% endif %}
 
-{% if site.twitter %}<a href="https://twitter.com/{{ site.twitter }}"
+{% if site.bluesky %}<a href="https://bsky.app/profile/{{ site.bluesky }}"
    target="_blank"
    rel="noopener "
   class="button button-icon">
-<svg xmlns="http://www.w3.org/2000/svg" height="32" width="32" viewBox="0 0 512 512"><!--!Font Awesome Free 6.5.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free Copyright 2023 Fonticons, Inc.--><path fill="#741654" d="M389.2 48h70.6L305.6 224.2 487 464H345L233.7 318.6 106.5 464H35.8L200.7 275.5 26.8 48H172.4L272.9 180.9 389.2 48zM364.4 421.8h39.1L151.1 88h-42L364.4 421.8z"/></svg>
-    <span class="screen-reader-text">Twitter</span>
+<svg xmlns="http://www.w3.org/2000/svg" height="32" width="32" viewBox="0 0 512 512"><!--!Font Awesome Free 6.7.2 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free Copyright 2025 Fonticons, Inc.--><path fill="#741654" d="M111.8 62.2C170.2 105.9 233 194.7 256 242.4c23-47.6 85.8-136.4 144.2-180.2c42.1-31.6 110.3-56 110.3 21.8c0 15.5-8.9 130.5-14.1 149.2C478.2 298 412 314.6 353.1 304.5c102.9 17.5 129.1 75.5 72.5 133.5c-107.4 110.2-154.3-27.6-166.3-62.9l0 0c-1.7-4.9-2.6-7.8-3.3-7.8s-1.6 3-3.3 7.8l0 0c-12 35.3-59 173.1-166.3 62.9c-56.5-58-30.4-116 72.5-133.5C100 314.6 33.8 298 15.7 233.1C10.4 214.4 1.5 99.4 1.5 83.9c0-77.8 68.2-53.4 110.3-21.8z"/></svg>
```

Comment:
I know this isn't necessarily the right PR for this, but does anyone know why we are using these complicated svgs with path, rather than the `fa-brands fa-bluesky`, something like `<i class="fab fa-bluesky" aria-hidden="true"></i>`. Do we not load the Font Awesome stylesheet at all and so the icons are not available?

Reply:
Ugh sorry, actually before we resolve this. I did notice we have one more file to consider. We are not using the svgs anywhere else on this website (which is fine). But more specifically, we do have a "share button" file we'll need to add fa-bluesky to, to replace twitter. _includes/buttons/share-buttons.html. I need to check to make sure how this is used, because we likely need to update the link for this as well...

Reply:
> Ugh sorry, actually before we resolve this. I did notice we have one more file to consider. We are not using the svgs anywhere else on this website (which is fine). But more specifically, we do have a "share button" file we'll need to add fa-bluesky to, to replace twitter. _includes/buttons/share-buttons.html. I need to check to make sure how this is used, because we likely need to update the link for this as well...

I mentioned this in the Slack channel, and I was kinda thinking that it's a separate issue, since the share buttons have other things that aren't in the footer now, like Facebook and Reddit.  I'm totally OK with changing them in this PR though.